src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss

```diff
@@ -123,39 +123,77 @@ $round-image-width: 220px;
   font-weight: 500;
 }
 
+.subInfoWrapper {
+  @include shadow-bottom;
+
+  background-color: $color-primary--020;
+}
+
 .subInfoContainer {
   @include container;
 
-  display: flex;
-  flex-direction: column;
   padding-top: 42px;
   padding-bottom: 42px;
+}
 
-  @include for-tablet-portrait-up {
-    flex-direction: row;
+.legend {
+  margin: 40px 0 12px;
+  font-size: $font-size--default;
+  font-weight: $font-weight--default;
+
+  @include for-tablet-landscape-up {
+    display: flex;
+    flex-flow: row wrap;
   }
 }
 
-.subInfoTextWrapper {
-  flex: auto;
-  margin-bottom: 32px;
+.term {
+  font-weight: $font-weight--heavier;
+  margin: 0 0 10px;
 
-  @include for-tablet-portrait-up {
-    margin-bottom: 0;
-    margin-right: 64px;
+  @include for-tablet-landscape-up {
+    flex: 1 0 26%;
```

Comment:
You should generally never need to set basis directly (the spec recommends against it). Instead just set the width. That way you can just define the width of the title (with `flex: none` set which is nice and obvious) and have the description fill the rest (with `flex: auto`) so you don't need to set the width manually for these.

Also, since you essentially want a consistent gap between the right end of "Performance" and the description, you can simplify even further by just setting the width and you don't even need the breakpoints. So:

```scss

.term {
  font-weight: $font-weight--heavier;
  margin: 0 0 10px;
  flex: none;

  @include for-tablet-landscape-up {
    width: 200px;
    margin-bottom: 20px;

    &:nth-last-child(2) {
      margin-bottom: 0;
    }
  }
}

.definition {
  margin: 0 0 30px;
  flex: auto;

  &:last-child {
    margin-bottom: 0;
  }

  @include for-tablet-landscape-up {
    margin-bottom: 20px;
  }
}
```

And if you wrap each row in a `<div>` you can do away with the clunky `&:nth-last-child(2)` styles.